src/Judge/NTS.Judge.Blazor/Core/Rankings/Protocols/Protocol.razor

```diff
@@ -7,68 +7,66 @@
 }
 <MudStack Row StretchItems="StretchItems.Middle" AlignItems="AlignItems.Center">
     <MudImage Height="120" Width="135" Src="images/logo-nak.jpg" />
-    
-    <MudStack>
-        <NHeadline2 Content="@Header.Title" />
 
-        <MudStack Row Class="d-flex" Justify="Justify.Center">
-            <NHeadline2 Content="@Header.PopulatedPlace.City" />
-
-            <NHeadline2 Content="@Header.EventSpan.ToString()" />
+    <MudImage Height="120" Width="135" Src="images/logo-bfks.png" />
+</MudStack>
+<MudStack>
+    <NHeadline2 Content="@Header.Title" />
 
-            <NHeadline2 Content="@Header.PopulatedPlace.Country.ToString()" />
+    <MudStack Row Class="d-flex" Justify="Justify.Center">
+        <NHeadline2 Content="@Header.PopulatedPlace.City" />
+        <NHeadline2 Content="@Header.EventSpan.ToString()" />
+        <NHeadline2 Content="@Header.PopulatedPlace.Country.ToString()" />
+    </MudStack>
+    <MudItem Class="d-flex flex-row justify-center mt-n5">
+        <MudText Typo="Typo.overline" Style="font-size:20px;" Class="printable">
+            @Officials_string:
+        </MudText>
+    </MudItem>
+    <MudStack Class="d-flex mb-3" Row="true" Justify="Justify.Center">
+        <MudStack Row="false">
+            @if (Header.GroundJuryPresident != null)
+            {
+                <MudText Typo="Typo.caption">
+                    Header.GroundJuryPresident
```

Comment:
These should be `@Header.GroundJuryPresident`. Right now they will probably be treated as plain text. Also you can put them on one line, as they are short